examples/remix-auth-email-link/README.md

```diff
@@ -0,0 +1,18 @@
+# Remix Auth Email Link
+
+Passwordless authentication by sending a magic link to your email. This example is using [remix-auth](https://github.com/sergiodxa/remix-auth) and [remix-auth-email-link](https://github.com/pbteja1998/remix-auth-email-link) packages.
+
+## Preview
+
+Open this example on [CodeSandbox](https://codesandbox.com):
+
+[![Open in codesandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/remix-run/remix/tree/main/examples/remix-auth-email-link)
+
+## Example
+
+This is using `remix-auth` and `remix-auth-email-link` packages. Instead of actually sending the magic link to the email, in this example I am just logging it on to the console. You can take care of that implementation yourself.
```

Comment:
```suggestion
This is using `remix-auth` and `remix-auth-email-link` packages. Instead of actually sending the magic link to the email, in this example I am just logging it on to the console. You can then copy/paste that into the URL. For sending the email you'll likely want to use an email service like SendGrid or Mailgun.
```